spec/core/ics-004-channel-and-packet-semantics/UPGRADES.md

```diff
@@ -617,9 +624,9 @@ function cancelChannelUpgrade(
     // get current sequence
     // If counterparty sequence is less than the current sequence, abort transaction since this error receipt is from a previous upgrade
     // Otherwise, set the sequence to counterparty's error sequence+1 so that both sides start with a fresh sequence
-    currentSequence = provableStore.get(currentSequencePath(portIdentifier, channelIdentifier))
```

Comment:
I think this `currentSequencePath` was a typo... It should have been `upgradeSequencePath`, right?